source/includes/fact-non-utf8-data.rst

```diff
@@ -0,0 +1,17 @@
+If Compass encounters data display issues when querying your data 
+with the error message: ``Invalid UTF-8 string in BSON document``. You 
+can disable UTF8 validation by setting the ``enableUtf8Validation`` 
+URI option to ``false``. 
```

Comment:
(question/context) 
What are the ramifications of disabling UTF-8 validation. Do we "recommend" that you do this in all/some situations?

Reply:
I added a warning callout, that this should be a temporarily method.